ftw/linkchecker/linkchecker.py

```diff
@@ -0,0 +1,100 @@
+from multiprocessing import Pool
+import time
+import requests
+import urls_testing
```

Comment:
Please do not use relative imports; use absolute imports.
`from ftw.linkchecker import urls_testing`

Reply:
👍